packages/terra-form-select/src/clinical-theme/NativeSelect.module.scss

```diff
@@ -0,0 +1,65 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .clinical-theme {
```

Comment:
⚠️ Remember to remove `clinical-theme` and keep internal before final.

Reply:
Removed.